src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferredOutEvidenceViewModelMap.cs

```diff
@@ -22,7 +22,11 @@ public TransferredOutEvidenceNotesSchemeViewModel Map(TransferredOutEvidenceNote
 
             foreach (var evidenceNoteRowViewModel in model.EvidenceNotesDataList)
             {
-                evidenceNoteRowViewModel.DisplayViewLink = evidenceNoteRowViewModel.Status.Equals(NoteStatus.Draft) || evidenceNoteRowViewModel.Status.Equals(NoteStatus.Submitted) || evidenceNoteRowViewModel.Status.Equals(NoteStatus.Approved);
+                evidenceNoteRowViewModel.DisplayViewLink = evidenceNoteRowViewModel.Status.Equals(NoteStatus.Draft) ||
```

Comment:
I think we can probably get rid of the setting of DisplayViewLink now as it should just show all Notestatus...  and remove the DisplayViewLink check from the view.

Reply:
done!